### DISCUSSION TITLE
Rotate gphoto2 capture and live view

### DISCUSSION THREAD

**Discussion by LordGith** [photobooth-app/photobooth-app#381] [Q&A]
Hi,
I've just switched over from a different photobooth app.
Due to constraints of my photobooth box, I've put the camera upside down (Canon EOS 70D).
Is it possible to rotate the image from gphoto2?
I only see the option for OpenCV2

Thank you!

**Comment by mgineer85** [comment]
Hi, its only implemented for backends that support it natively. 
but I think it would be a good addition.

**Reply by LordGith** [reply]
Hi Michael, How difficult would it be to implement this?
I'm looking for a solution by tomorrow :/ If to difficult, I will try to find an alternative solution.

**Reply by mgineer85** [reply]
Hi - I implemented it right now in v5 which is available as beta.
You could just give it a try, but it might fail because there are many changes to v5.

Depending on your installation method, you need to explicitly install the beta:
`pip install photobooth-app==5.0.0b3`

After installation each backend has an "orientation" paramter, that can be used to flip and rotate the images.

If you do not want to go beta, I think there is no solution for tomorrow except from turning the camera.

**Reply by LordGith** [reply]
Hi Michael,
This is working perfectly for me!
V5 looks amazing. I'll send you a picture of my photobooth this weekend after the event.
1 extra request if I may, Being able to change the background around the picture. Now it is dull grey, and quite harsh in low light environments. The same for the main page.
Thank you for your hard work on this project!

**Reply by mgineer85** [reply]
You could set the system/browser to darkmode and the site would change. 
otherwise you can change any css using the private css.
